src/content/en/2020/media.md

```diff
@@ -23,3 +24,439 @@ queries: 05_Media
+
+##### Picture: format switching
+
+While there are some services and image CDNs which can provide auto-format switching from a single image reference, we can also achieve similar behaviours using standard markup with the picture element.
+
+```html
+<picture>
+    <source type="image/webp" srcset="images/example.webp" />
+    <img src="images/example.jpg" alt="..." />
+</picture>
+```
+
+Current uptake shows around 19% of pages being served using the picture element are doing so at least in part in relation to format switching.
+
+{{ figure_markup(
+  image="use-of-picture.png",
+  alt="Use of picture.",
+  caption="Use of `<picture>`.",
+  description="Stacked bar chart showing 19.30% of pages use `<picture>` on desktop and 80.70% do not. On mobile it's similar with 18.54% using and 81.46% not using.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTLNnD9VNqXNxMu60VovxIEp_L6vmNo1oWt8-C18DOetXB3qIkee_-KjZwYYPIkkIM-7So-5wBwQ4QY/pubchart?oid=416496535&format=interactive",
+  sheets_gid="1719719920",
+  sql_file="picture_format_distribution.sql"
+  )
+}}
+
+Breaking this down into the number of formats offered:
+
+{{ figure_markup(
+  image="picture-number-of-formats.png",
+  alt="picture number of formats.",
+  caption="`<picture>` number of formats.",
+  description="Bar chart showing number of formats in picture usage. 68.01% on desktop (68.03% on mobile) use 1 format. 2 formats is 23.78% and 23.78% respectively, 3 formats is 7.00% and 6.97% and 4+ formats is only used on 1.21% and 1.22% pages respectively.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTLNnD9VNqXNxMu60VovxIEp_L6vmNo1oWt8-C18DOetXB3qIkee_-KjZwYYPIkkIM-7So-5wBwQ4QY/pubchart?oid=1963933588&format=interactive",
+  sheets_gid="1719719920",
+  sql_file="picture_format_distribution.sql"
+  )
+}}
+
+Of pages using picture for format switching, around 68% are offering a single type variation, in addition to the img src which acts as the default.
+
+{{ figure_markup(
+  image="picture-format-usage-by-type.png",
+  caption="Picture format usage by type.",
+  description="Bar chart showing picture format usage by image type. WebP is dominate usage used by 83.29% of picture elements on desktop (84.64% on mobile). Next is JPG used by 4.84% and 4.83% respectively, followed by PNG at 18.18% and 17.46%, Gif at 0.53% and 0.53% and AVIF doesn't register above 0 for either client.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTLNnD9VNqXNxMu60VovxIEp_L6vmNo1oWt8-C18DOetXB3qIkee_-KjZwYYPIkkIM-7So-5wBwQ4QY/pubchart?oid=775091522&format=interactive",
+  sheets_gid="1719719920",
+  sql_file="picture_format_distribution.sql"
+  )
+}}
+
+{# TODO Analysts this doesn't sound right? Is this the format chosen? That would make sense as we crawl as Chrome so it will prefer WebP over others, but if not then I'm very dubious about these stats. Can we double check? #}
```

Comment:
```suggestion
{{ figure_markup(
  image="picture-format-usage-by-type.png",
  caption="Picture format usage by type.",
  description="Bar chart showing picture format usage by image type. WebP is dominate usage used by 83.29% of picture elements on desktop (84.64% on mobile). Next is JPG used by 4.84% and 4.83% respectively, followed by PNG at 18.18% and 17.46%, Gif at 0.53% and 0.53% and AVIF doesn't register above 0 for either client.",
  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTLNnD9VNqXNxMu60VovxIEp_L6vmNo1oWt8-C18DOetXB3qIkee_-KjZwYYPIkkIM-7So-5wBwQ4QY/pubchart?oid=775091522&format=interactive",
  sheets_gid="1719719920",
  sql_file="picture_format_distribution.sql"
  )
}}

{# TODO Analysts this doesn't sound right? Is this the format chosen? That would make sense as we crawl as Chrome so it will prefer WebP over others, but if not then I'm very dubious about these stats. Can we double check? #}
```

@smatei can you look into this?

Reply:
@smatei / @tpiros I added this text below the graph:

```md
We see that WebP is the dominante usage across `<picture>` elements, followed by PNG and JPG is only 4.83% of `<picture>` usage.
 
<p class="note">Note our crawler crawls as Chrome which supports WebP, but if using anoither browser which does not support this then you will see different results.</p>
```